src/components/button/button-small/button-small.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { Button } from '../button';
+import { ButtonSharedProps } from '../button-shared.props';
+import { useButtonSmallStyles } from './button-small.styles';
+
+interface Props extends Omit<ButtonSharedProps, 'iconName'> {
+  title: string;
+}
+
+export function ButtonSmall(props: Props) {
```

Comment:
let's use arrow functions everywhere and `FC<Props>` type to make correct return from component